getRepos/gitstuff.bat

```diff
@@ -0,0 +1,556 @@
+@echo off 
+SETLOCAL EnableDelayedExpansion 
+ 
+::########################################################### 
+::#               											# 
+::# Filename:  getLoma.bat         							# 
+::#               											# 
+::# Description:  Configures the LOMA dev env for SW ENG 	# 
+::#               											# 
+::# History:            									# 
+::#               											# 
+::# Date  		Note       			SW ENG  				# 
+::# 20190522 	Initial Release     BMD   					# 
+::#               											# 
+::########################################################### 
+ 
+::---------------- 
+::-  VAR REGION  - 
+::---------------- 
+ 
+:: Err Message strings 
+set errorMessage[000]=Failed to receive proper err code, review %log% and contact SIT OKC point of contact if additional support is required. 
```

Comment:
Why are we pre setting all of these err messages?

Reply:
I was wondering the same thing myself.

Reply:
We set these err msgs for ease of maintainability for future err msgs and being all defined top level.